Formula/miniserve.rb

```diff
@@ -0,0 +1,18 @@
+class Miniserve < Formula
+  desc "High performance static file server"
+  homepage "https://github.com/svenstaro/miniserve"
+  url "https://github.com/svenstaro/miniserve/archive/v0.1.5.tar.gz"
+  sha256 "b435471cd08f3dbe080e26a0b6937c824502a87659948f49b5d2e54f07de64e0"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix
+  end
+
+  test do
+    # miniserve has no options that do not immediately start listening on ports
```

Comment:
As @DomT4 said, we need a test that exercises the some of the functionality of the app. Look at `woof` for a very simple example.

Reply:
Thanks for the direction. `woof` did almost exactly what I wanted to do but didn't know if that was acceptable.